source/CIGameScene.cpp

```diff
@@ -390,3 +410,20 @@ void GameScene::processSpecialStardust(const cugl::Size bounds, const std::share
     
     stardustQueue->clearPowerupQueue();
 }
+
+/**
+ * Sets whether the pause menu is currently active and visible.
+ *
+ * @param onDisplay     Whether the pause menu is currently active and visible
+ */
+void GameScene::togglePause(bool onDisplay) {
```

Comment:
i think the pause button should be in pause menu so we don't need this method in game scene